package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Rentzila",
+  "type": "module",
+  "scripts": {
+    "wdio-all": "npx wdio run ./wdio.conf.ts",
+    "wdio-authorization": "npx wdio run ./wdio.conf.ts --suite authorization"
+  },  
+  "devDependencies": {
+    "@wdio/allure-reporter": "^9.0.8",
+    "@wdio/cli": "^9.0.9",
```

Comment:
@Anastas11aZakharova Please, update the dependencies to the latest versions (only in cases with your own repo), or, add the info to the README or description to the PR about the node version used.

Reply:
Fixed in the latest commit